packages/tools/fluid-runner/src/fluidRunner.ts

```diff
@@ -58,13 +64,25 @@ function fluidRunner() {
                 const logger = ChildLogger.create(fileLogger, "LocalSnapshotRunnerApp",
                     { all: { Event_Time: () => Date.now() } });
 
-                const result = await exportFile(
-                    argv.codeLoader,
-                    argv.inputFile,
-                    argv.outputFolder,
-                    argv.scenario,
-                    logger,
-                );
+                let result: IExportFileResponse;
+                // codeLoader argument always takes precedence
+                if (argv.codeLoader) {
+                    result = await parseBundleAndExportFile(
+                        argv.codeLoader,
+                        argv.inputFile,
+                        argv.outputFile,
+                        logger,
+                        argv.options,
+                    );
+                } else {
+                    result = await exportFile(
+                        fluidFileConverter!,
```

Comment:
what if we have both fluidFileConverter and argv.codeLoader?

Reply:
`argv.codeLoader` will take precedence. This is noted in the comment above the `if` and in the README.md

Reply:
But is this expected usage pattern? If not, I'd disallow it.